pages/data-design/naming-conventions.mdx

```diff
@@ -72,7 +68,7 @@ Having a global namespace for all events and properties significantly reduces du
    For example: Avo won't allow you create a `signupStart` event if `SIGNUP_START` already exists:
 
    <img
-     src={require('../../images/best-practices/naming-conventions/already_exists.png')}
+     src={'../../images/best-practices/naming-conventions/already_exists.png'}
```

Comment:
More for setting the example:
```suggestion
     src={'/images/best-practices/naming-conventions/already_exists.png'}
```

and those below. Don’t know why this works though 🤔